tests/Fixer/ArrayNotation/MultilineLongArrayFixerTest.php

```diff
@@ -0,0 +1,437 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Tests\Fixer\ArrayNotation;
+
+use PhpCsFixer\Tests\Test\AbstractFixerTestCase;
+
+/**
+ * @author Philippe Bouttereux <philippe.bouttereux@gmail.com>
+ *
+ * @internal
+ *
+ * @covers \PhpCsFixer\Fixer\ArrayNotation\MultilineLongArrayFixer
+ */
+final class MultilineLongArrayFixerTest extends AbstractFixerTestCase
+{
+    /**
+     * @param array<string, mixed> $config
+     *
+     * @dataProvider provideFixCases
+     */
+    public function testFix(string $expected, ?string $input = null, array $config = []): void
+    {
+        $this->fixer->configure($config);
+
+        $this->doTest($expected, $input);
+    }
+
+    public static function provideFixCases(): array
+    {
+        return [
+            // Old style array
```

Comment:
Please use `yield 'old style array' => [ ... ];` for each case (requires `iterable` return type instead of an `array`). It's better from performance point of view (memory usage), but also cases' descriptions can be displayed when running with `--testdox` option. [See existing provider example](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/blob/12792fa271ad216530a7916fca91e397f7fe1ff4/tests/Fixer/Alias/ArrayPushFixerTest.php#L36-L39).

Reply:
Same mistake there, I think we were both biased by the Cookbook which returns an array